issue with Travis deploy with gradle

failure: https://travis-ci.org/ibm-functions/runtime-nodejs/builds/276322251
```
FAILURE: Build failed with an exception.
* Where:
Build file '/home/travis/build/ibm-functions/runtime-nodejs/tests/build.gradle' line: 23
* What went wrong:
A problem occurred evaluating project ':tests'.
> Could not get unknown property 'sourceSets' for project ':whisktests' of type org.gradle.api.Project.
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
* Get more help at https://help.gradle.org
BUILD FAILED in 6s

```

--- a/tools/travis/build.sh
+++ b/tools/travis/build.sh
@@ -6,6 +6,7 @@ set -ex
 SCRIPTDIR=$(cd $(dirname "$0") && pwd)
 ROOTDIR="$SCRIPTDIR/../.."
 WHISKDIR="$ROOTDIR/../openwhisk"
+IMAGE_PREFIX="testing"
 
 export OPENWHISK_HOME=$WHISKDIR
 
@@ -18,7 +19,6 @@ TERM=dumb ./gradlew \
 
 # Build OpenWhisk
 cd $WHISKDIR
-IMAGE_PREFIX="testing"
 #superfast option
 docker pull openwhisk/controller
 docker tag openwhisk/controller ${IMAGE_PREFIX}/controller

--- a/tools/travis/build.sh
+++ b/tools/travis/build.sh
@@ -7,16 +7,17 @@ SCRIPTDIR=$(cd $(dirname "$0") && pwd)
 ROOTDIR="$SCRIPTDIR/../.."
 WHISKDIR="$ROOTDIR/../openwhisk"
 
+export OPENWHISK_HOME=$WHISKDIR
+
 # Build IBM nodejs runtime
-cd $ROOTDIR/8.5
-docker build . -t action-nodejs-ibm-v8.5
-docker tag action-nodejs-ibm-v8.5 testing/action-nodejs-ibm-v8.5
+cd $ROOTDIR
+TERM=dumb ./gradlew \
+:8.5:distDocker \
+-PdockerImagePrefix=${IMAGE_PREFIX}
 
 
 # Build OpenWhisk
-
 cd $WHISKDIR
-export OPENWHISK_HOME=$WHISKDIR
 IMAGE_PREFIX="testing"
 #superfast option
 docker pull openwhisk/controller

--- a/tools/travis/deploy.sh
+++ b/tools/travis/deploy.sh
@@ -5,6 +5,7 @@ set -eu
 
 SCRIPTDIR=$(cd $(dirname "$0") && pwd)
 ROOTDIR="$SCRIPTDIR/../.."
+WHISKDIR="$ROOTDIR/../openwhisk"
 
 dockerhub_image_prefix="$1"
 dockerhub_image_tag="$2"
@@ -12,6 +13,7 @@ dockerhub_image_tag="$2"
 docker login -u "${DOCKER_USER}" -p "${DOCKER_PASSWORD}"
 
 cd ${ROOTDIR}
+export OPENWHISK_HOME=$WHISKDIR
 TERM=dumb ./gradlew :8.5:distDocker \
 -PdockerImagePrefix=${dockerhub_image_prefix} 
 -PdockerImageTag=${dockerhub_image_tag}

